wiki.py

```diff
@@ -1,10 +1,10 @@
 import logging
 import re
 
-from pywikibot import Site
-from pywikibot import Page
-from wikitables.util import ftag
 import mwparserfromhell as mwp
+from wikitables.util import ftag
+
+from pywikibot import Page, Site
```

Comment:
Is having from...imports on one line enforced by anything? PEP 8 seem to allow both. Personally, I prefer separate lines.

Reply:
It's the default in isort, which I interpret as meaning it's the default interpretation of PEP8. I.e. single line for normal imports but group them when it's `from X import ...`

If you have strong opinions about it I can add `force_single_line = true` to the tox but this (grouped `from X import ...`) is also how I would expect to see it

Reply:
Let's keep it like this if it's default in the checkers.